src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -480,31 +53,59 @@
     }
 
     &:hover {
-      color: var(--navbar-active);
+      color: var(--);
     }
   }
 
   .topRowWrapper {
     display: flex;
-    grid-area: top;
-    gap: var(--space-sm);
-    z-index: 30;
+    grid-area: header;
+    align-items: center;
+    justify-content: space-between;
+    gap: auto;
+    z-index: 1;
+    width: 100%;
 
     & > * {
       filter: drop-shadow(4px 4px 8px rgba(0, 0, 0, 0.25));
     }
 
     .backButton {
-      margin-inline-end: auto;
+      display: flex;
+      justify-content: flex-start;
+      gap: var(--space-sm);
+      align-self: start;
```

Comment:
something in this new style is making the arrow uncentered in the button

![image](https://github.com/user-attachments/assets/ecebc8a7-7412-4440-a46b-3a789dad4340)

Reply:
Thanks for the tips! I will return back to this in next weekend to polish it more and style the tabs for the ExtraInfoWrapper a little more too (it's when I have more free time). After that I will request review from you, Flavio and the other devs! 😊